.env.d.ts

```
@@ -1,7 +1,6 @@
-##EX
-declare module "@env" {
+declare module '@env' {
```

dotenv-linter (3.3.0)

[warning] [IncorrectDelimiter] The declare module '@env' { key has incorrect delimiter

(IncorrectDelimiter)